docs/public/static/diffs/template-bare-minimum/raw/50..51.diff

```diff
@@ -313,25 +313,37 @@ index b3978b9515..eaf07ca4c2 100644
  target 'HelloWorld' do
    use_expo_modules!
    config = use_native_modules!
-@@ -49,8 +25,6 @@ target 'HelloWorld' do
+@@ -49,15 +25,17 @@ target 'HelloWorld' do
      :hermes_enabled => podfile_properties['expo.jsEngine'] == nil || podfile_properties['expo.jsEngine'] == 'hermes',
      # An absolute path to your application root.
      :app_path => "#{Pod::Config.instance.installation_root}/..",
 -    # Note that if you have use_frameworks! enabled, Flipper will not work if enabled
 -    :flipper_configuration => flipper_config
++    # Temporarily disable privacy file aggregation by default, until React
++    # Native 0.74.2 is released with fixes.
```

Comment:
nit: to generate the diff again and pull out the comment

Reply:
I've tried but it's not removing it 🤔